catalogue/webapp/utils/search.ts

```diff
@@ -1,16 +1,53 @@
+import {
+  CatalogueApiError,
+  CatalogueResultsList,
+  Image,
+  Work,
+} from '@weco/common/model/catalogue';
+import { Story } from '@weco/catalogue/services/prismic/types/story';
+import {
+  PrismicApiError,
+  PrismicResultsList,
+} from '@weco/catalogue/services/prismic/types';
+
 export type DefaultSortValuesType = {
   sort: string | undefined;
   sortOrder: string | undefined;
 };
 
-//  SORT
+type QueryResultsProps = (Work | Image | Story)[] | undefined;
+
+/**
+ * Takes query result and checks for errors to log before returning required data.
+ * @param {string} categoryName - e.g. works
+ * @param queryResults - Original result from query
+ */
+export const setQueryResultsData = (
```

Comment:
nit: this function doesn't set anything, it extracts a field from a value.

`getQueryResults`?